client/src/App.js

```diff
@@ -21,11 +21,16 @@ import {
   faAngleLeft,
   faAngleRight,
   faCommentDots,
-  faEdit,
+  faEdit
 } from "@fortawesome/free-solid-svg-icons";
 import { fab } from "@fortawesome/free-brands-svg-icons";
 import { fas } from "@fortawesome/free-solid-svg-icons";
 import { far } from "@fortawesome/free-regular-svg-icons";
+import { BASE_URL } from "./services/constants";
+import { handleSignup } from "./services/User";
+import { handleLogin } from "./services/User";
+import { handleLogout } from "./services/User";
```

Comment:
@natesanchez FYI you can merge these:

```js
import { handleSignup, handleLogin, handleLogout } from "./services/User";
```

Reply:
@ZakkMan sure thing, I think I know what it could be